examples/with-nextjs/ponder/ponder.schema.ts

```diff
@@ -1,4 +1,4 @@
-import { onchainTable } from "ponder";
+import { onchainTable } from "ponder/schema";
```

Comment:
Honestly I could see us removing `onchainTable` from the root entrypoint so users get the tree-shaking benefit for free. Tiny breaking change in the grand scheme of things.

Do worry a bit about `ponder/schema` vs `ponder:schema` though ...